### PR TITLE
add easyconfig file for Vampir

### PR DESCRIPTION
**Pull request by boegel**
this easyconfig is contributed by @berndmohr via https://github.com/fgeorgatos/easybuild.experimental/tree/master/users/unite

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/v/Vampir/Vampir-8.1.0-linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+description = """The VAMPIR software tool provides an easy-to-use framework that enables
+ developers to quickly display and analyze arbitrary program behavior at any level of detail.
+ The tool suite implements optimized event analysis algorithms and customizable displays that
+ enable fast and interactive rendering of very complex performance monitoring data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['vampir-%(version)s-standard%(versionsuffix)s-setup.bin']
+
+install_cmd = "./vampir-%(version)s-standard%(versionsuffix)s-setup.bin --silent --instdir=%(installdir)s"
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/v/Vampir/Vampir-8.1.0-linux-x86_64.eb]: this has been reviewed some time ago; 
I still recommend the following construct, for compactness/maintainability:
`install_cmd = "./" + sources[0] + " --silent --instdir=%(installdir)s"`
re-testing recommended.

Comment (boegel) [inline easybuild/easyconfigs/v/Vampir/Vampir-8.1.0-linux-x86_64.eb]: additional options need to be passed to the install command as well; @geimer is looking into it

**Comment by boegel** [comment]
requires https://github.com/boegel/easybuild-easyconfigs/pull/8 (thanks @geimer!)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ba35119285c94bf48730 for a full test report.

**Comment by boegel** [comment]
Good to go, thanks for picking this up again @geimer!

**Pull request merged**